justfile

```diff
@@ -25,7 +25,11 @@ python-run file='':
 
 # Run the tests.
 test:
-        @py.test tests
+        @py.test tests --benchmark-skip
```

Comment:
You can remove this part, since I'm working on a new command `just benchmark`. Thanks!